scripts/generate-f32-gemm.sh

```diff
@@ -621,6 +621,11 @@ tools/xngen src/f32-gemm/avx512-broadcast.c.in -D MR=7 -D NR=16 -D INC=1 -D DATA
 tools/xngen src/f32-gemm/avx512-broadcast.c.in -D MR=8 -D NR=16 -D INC=0 -D DATATYPE=F32 -o src/f32-gemm/gen/f32-gemm-8x16-minmax-avx512f-broadcast.c &
 tools/xngen src/f32-gemm/avx512-broadcast.c.in -D MR=8 -D NR=16 -D INC=1 -D DATATYPE=F32 -o src/f32-gemminc/gen/f32-gemminc-8x16-minmax-avx512f-broadcast.c &
 
+################################ RISC-V Vector ################################
+tools/xngen src/f32-gemm/MrxLmulv-rvv.c.in -D MR=7 -D LMUL=4 -D ACTIVATION=LINEAR -D DATATYPE=F32 -o src/f32-gemm/gen/f32-gemm-7x4v-rvv.c &
```

Comment:
LINEAR and RELU are low priority.  Only MINMAX is used typically.
They are used on WASMSIMD because MIN and MAX are very slow and a RELU implements faster than general purpose MAX(0,value).
MINMAX is actually not needed on LLMs so it may be interesting to revisit this.
On JIT we start with minmax code and remove the min and max when the code is generated.
But on precompiled code, having 3 versions created code bloat and is not affecting performance substantially.

Reply:
> They are used on WASMSIMD because MIN and MAX are very slow and a RELU implements faster than general purpose MAX(0,value).
MINMAX is actually not needed on LLMs so it may be interesting to revisit this.

I do not fully understand the meaning.
LLM doesn't need the minmax version of GEMM, so it is supposed to mean the Linear version is important for LLM.